src/js/components/GroupNav/GroupNav.js

```diff
@@ -1,23 +1,30 @@
 import React, { Component } from "react";
 import '../../../scss/components/GroupNav/GroupNav.scss';
+import GroupNavDMButtons from "../GroupNavDMButtons/GroupNavDMButtons";
 
 class GroupNav extends Component {
   render() {
     return(
       <div className="nav-column nav-column-message">
         <div className="nav-group nav-group-top">
-          <div className="nav-item-start">
-            <button className="btn btn-start">Find or start a conversation</button>
+          <div className="nav-item nav-item-start">
+            <button className="nav-item-start-content">Find or start a conversation</button>
           </div>
         </div>
-        <div className="nav-group">
-          <button className="nav-item nav-item-menu">Friends</button>
-          <button className="nav-item nav-item-menu">Nitro</button>
-        </div>
-        <div className="nav-group nav-group-grow">
-          <div className="nav-item-head">Direct Messages</div>
-          <button className="nav-item nav-item-dm">Pytho</button>
-          <button className="nav-item nav-item-dm">Jakx</button>
+        <div className="nav-group nav-group-dm flex-grow">
+          <div className="nav-subgroup nav-subgroup-menu">
+            <div className="nav-item nav-item-menu">
+              <button className="nav-item-btn">
+                <div className="nav-item-btn-text">Friends</div>
+              </button>
```

Comment:
`nav-item` and `nav-item-btn` are so similar that this reads to me like `-btn` is an extension of `nav-item`, can this convention be a little more defined?